deployer/deployer.go

```diff
@@ -137,7 +161,12 @@ func (d *DeployerImpl) deploy(
 			}
 			log.Printf("Number of public ips: %d\n", publicIPCount)
```

Comment:
please make sure the logs are based on a level e.g trace instead or remote all together